timescaledb/overview/release-notes/index.md

```diff
@@ -14,6 +14,43 @@ Want to stay up-to-date with new releases? Subscribe to get
 [release notes email updates](https://www.timescale.com/signup/release-notes).
 </highlight>
 
+## 2.10.0 (2023-02-21)
+
+This release contains new features and bug fixes since the 2.9.3 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+* Joins in continuous aggregates
+* Re-architecture of how compression works: ~2x improvement on INSERT rate into compressed chunks.
+* Full PostgreSQL 15 support for all existing features. Support for the newly introduced MERGE command on hypertables will be introduced on a follow-up release.
+
+**PostgreSQL 12 deprecation announcement**
+We will continue supporting PostgreSQL 12 until July 2023. Sooner to that time, we will announce the specific version of TimescaleDB in which PostgreSQL 12 support will not be included going forward.
```

Comment:
```suggestion
We will continue supporting PostgreSQL 12 until July 2023. Before that time, we will announce the specific version of TimescaleDB in which PostgreSQL 12 support will be removed.
```

Reply:
Changed

Reply:
@jnidzwetzki I don't see this change on the latest version of the release notes

Reply:
@iroussos Thank you for pointing this out. I have forgotten to push this change.